pkg/base/config.go

```diff
@@ -55,7 +55,39 @@ const (
 
 	// defaultRangeLeaseRaftElectionTimeoutMultiplier specifies what multiple the
 	// leader lease active duration should be of the raft election timeout.
-	defaultRangeLeaseRaftElectionTimeoutMultiplier = 3
+	//
+	// Timers for Raft leadership election and lease expiration run in parallel.
+	// Although not required, we would like to elect a leader before the lease
+	// expires, such that we don't have to wait for a Raft election when we're
+	// ready to acquire the lease.
+	//
+	// The relevant operations and default time intervals are (assuming RTTs
+	// ranging from 10ms to 400ms):
+	//
+	// Raft election:
+	// - Heartbeat offset (0-1 heartbeat interval)   [-1.00s - 0.00s]
```

Comment:
Could you elaborate on these a little bit more? I'm not sure what these "offsets" mean.
A summary line that shows the [min,max] duration would also be helpful.

Reply:
Sure, added a paragraph on it.